snap/snapcraft.yaml

```diff
@@ -47,6 +48,7 @@ parts:
         TRIPLET_PATH="$SNAPCRAFT_PART_INSTALL/usr/lib/$(gcc -print-multiarch)"
         LIBSODIUM=$(readlink -n $TRIPLET_PATH/libsodium.so.18)
         ln -s $LIBSODIUM $TRIPLET_PATH/libsodium.so
+        patch -d $SNAPCRAFT_PART_INSTALL/lib/python3.6/site-packages -p1 < $SNAPCRAFT_STAGE/pyyaml-support-high-codepoints.diff
     after: [python, apt]
```

Comment:
You probably want to add `patches` to after here.

Reply:
the after for python includes this, so the chain is somewhat complete; I will sort of ignore this as we are going back to `stage-packages` as soon as our SRUed deb gets in with the fixes for classic

Reply:
Good catch otherwise ;-)